API/controller/app.js

```diff
@@ -113,7 +113,11 @@ app.use(express.static(path.join(__dirname, "../public")));
 
 app.use(
   session({
+<<<<<<< HEAD
+    secret: "j2xOm%iKqFOqqNoJ#6DUlWmP(*JVIe!V",
+=======
     secret: "(@6UiSfotyHSp8HdC0N7OjKCX*7HtMyO",
+>>>>>>> c7b2459499e31c517bf5af3b121f5ccd7a4a6b75
```

Comment:
You'll want to remove these merge conflict markers.

Reply:
+1